scripts-available/CDB_CartodbfyTable.sql

```diff
@@ -166,7 +166,7 @@ BEGIN
         AS seq INTO rec2;
 
       -- Reset sequence name
-      sql := Format('ALTER SEQUENCE %s RESTART WITH %d', rec2.seq::text, rec.max + 1);
+      sql := Format('ALTER SEQUENCE %s RESTART WITH %', rec2.seq::text, rec.max + 1);
```

Comment:
I don't think that code is correct
```
# select format('test %s %', 'foo', 42);
ERROR:  unterminated format specifier
```
Have you checked that the test actually fails without the fix? I don't think you're exercising that line with the test.

Have you checked there are no other %X wrong format specifiers?